contracts/WETH10.sol

```diff
@@ -2,20 +2,48 @@ pragma solidity 0.7.0;
 // Copyright (C) 2015, 2016, 2017 Dapphub // Adapted by Ethereum Community 2020
 
 contract WETH10 {
-    string public constant name = "Wrapped Ether";
-    string public constant symbol = "WETH";
-    uint8  public constant decimals = 18;
-    bytes32 public immutable DOMAIN_SEPARATOR;
-    bytes32 public immutable PERMIT_TYPEHASH = keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)");
+    string public constant name = "Wrapped Ether"; // Declares fixed WETH10 token name - matches canonical WETH9 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+    string public constant symbol = "WETH"; // Declares fixed WETH10 token symbol - matches canonical WETH9 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+    uint8  public constant decimals = 18; // Declares fixed WETH10 token unit scaling factor - matches canonical WETH9 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+    bytes32 public immutable DOMAIN_SEPARATOR; // ERC2612 permit() pattern - hash identifies WETH10 token contract
+    bytes32 public immutable PERMIT_TYPEHASH = keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)"); // ERC2612 permit() pattern - hash identifies WETH10 token function for signature
 
+    /**
+     * @dev Emitted when the allowance of a `spender` for an `owner` account WETH10 token is set by
+     * a call to {approve}. `value` is the new allowance.
+     */
     event  Approval(address indexed owner, address indexed spender, uint256 value);
+    
+    /**
+     * @dev Emitted when `value` WETH10 token are moved from one account (`from`) to
+     * another (`to`).
+     */
     event  Transfer(address indexed from, address indexed to, uint256 value);
 
+    /**
+     * @dev Returns the amount of WETH10 token owned by an account.
+     */
     mapping (address => uint256)                       public  balanceOf;
+    
+    /**
+     * @dev Returns the current ERC2612 nonce for an account. This value must be
+     * included whenever a signature is generated for {permit}.
+     *
+     * Every successful call to {permit} increases the account's nonce by one. This
+     * prevents a signature from being used multiple times.
+     */
     mapping (address => uint256)                       public  nonces;
+    
+    /**
+     * @dev Returns the remaining number of WETH10 token that an account (second) will be
+     * allowed to spend on behalf of another (first) through {transferFrom}. This is
+     * zero by default.
+     *
+     * This value changes when {approve} or {transferFrom} are called.
+     */
     mapping (address => mapping (address => uint256))  public  allowance;
 
-    uint256 private unlocked = 1;
+    uint256 private unlocked = 1; 
```

Comment:
Trailing whitespace. Please run `yarn lint:sol` to find linting errors before pushing.